pandas/tests/indexes/test_interval.py

```diff
@@ -255,6 +255,30 @@ def test_insert(self):
         pytest.raises(ValueError, self.index.insert, 0,
                       Interval(2, 3, closed='left'))
 
+    @pytest.mark.parametrize('data', [
```

Comment:
prob need some more tests for ``.insert`` with (None, np.nan, pd.NaT as the inserted value) generally